sabnzbd/assembler.py

```diff
@@ -169,6 +169,11 @@ def _assemble(nzf, path, dupe):
     decodetable = nzf.decodetable
 
     for articlenum in decodetable:
+        # Break if deleted during writing
+        if nzf.nzo.deleted:
```

Comment:
Good find, but is this enough?
Should you test for nzf.nzo.status in (Status....) ?